### DISCUSSION TITLE
Realloc Error in Create Seurat Object

### DISCUSSION THREAD

**Discussion by FernandaScar** [satijalab/seurat#5985] [Q&A]
Hello everyone!

I'm new at sc analysis, sorry if is this a basic question!
I've try to run this command to create a seurat object:

```
library(Seurat)
library(tibble)
library(magrittr)

# Read in data
data <- data.table::fread("~/Downloads/GSE93374_Merged_all_020816_BatchCorrected_LNtransformed_doubletsremoved_Data.txt.gz", data.table = F)

# Gene names are in the first column so we need to move them to rownames
data <- data %>% 
  column_to_rownames("V1")

# Create Seurat
seurat <- CreateSeuratObject(counts = data)
```
But I've got this error:
Error in asMethod(object) : 'R_Realloc' could not re-allocate memory (3304504368 bytes)

I thought it was because of my computer RAM memory, so I tried with a friend's computer with a better memory and I've got no luck with that.

Anyone who knows how to solve? It's a pretty large file, but I've seen other analysis with bigger files that didn't get this issue!

Hope someone can help me!

**Comment by timoast** [comment]
It looks like you don't have enough memory to store the matrix. It looks like that is a batch-corrected matrix, which is dense and so will require more memory than the original count matrix. I'd suggest trying the count matrix instead, which is also on GEO
